app/main.py

```diff
@@ -16,4 +16,6 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+sorted_variables = {"immutable": [lucky_number, pi, one_is_a_prime_number, name, profile_info],
```

Comment:
Correct your code according to the following structure
<img width="205" alt="Screenshot 2023-10-16 at 16 58 49" src="https://github.com/mate-academy/py-mutable-immutable/assets/117740672/1c1a865e-6756-48b5-8813-3e9f11bd221b">